modules/widgets/eu-cookie-law/widget.php

```diff
@@ -14,12 +22,12 @@ class="<?php echo implode( ' ', $classes ); ?>"
 		echo nl2br( esc_html( $instance['customtext'] ) );
 	} ?>
 
-	<a href="<?php
-		$policy_link_text = 'default' === $instance['policy-url'] || empty( $instance['custom-policy-url'] )
-			? $instance['default-policy-url']
-			: $instance['custom-policy-url'];
-		echo esc_url( $policy_link_text );
-	?>" >
+	<?php
+	$policy_link_url = 'default' === $instance['policy-url'] || empty( $instance['custom-policy-url'] )
+		? $instance['default-policy-url']
+		: $instance['custom-policy-url'];
+	?>
+	<a href="<?php echo esc_url( $policy_link_url ); ?>" rel="nofollow">
```

Comment:
Like above, should we only apply this when the link is pointing to Jetpack.com, and not some other custom link?